plugins/memcached/lib_memcached.ecllib

```diff
@@ -37,4 +37,6 @@ export memcached := SERVICE : plugin('memcached'), namespace('MemCachedPlugin')
   BOOLEAN Exists(CONST VARSTRING key, CONST VARSTRING options, CONST VARSTRING partitionKey = '') : cpp,action,context,entrypoint='MExists';
   CONST VARSTRING KeyType(CONST VARSTRING key, CONST VARSTRING options, CONST VARSTRING partitionKey = '') : cpp,action,context,entrypoint='MKeyType'; //NOTE: calls get
   Clear(CONST VARSTRING options) : cpp,action,context,entrypoint='MClear';
+  Delete(CONST VARSTRING key, CONST VARSTRING options, CONST VARSTRING partionKey = '') : cpp,action,context,entrypoint='MDelete';
```

Comment:
The redis plugin documents Del and not Delete (As Del is the actual redis command). You asked me to also add Delete such that an more intuitive form was present. You also asked me to add it such that it conformed with the version of the memcached function, which even I had forgotten that there wasn't one. This is it. Delete makes more sense for memcached and thus I have added Del for some ease of conformity of use between the two. Del will however not be documented.